src/settings-ui/Settings.UI.Library/VideoConferenceConfigProperties.cs

```diff
@@ -74,8 +72,8 @@ public VideoConferenceConfigProperties()
         [JsonPropertyName("theme")]
         public StringProperty Theme { get; set; }
 
-        [JsonPropertyName("hide_toolbar_when_unmuted")]
-        public BoolProperty HideToolbarWhenUnmuted { get; set; }
+        [JsonPropertyName("toolbar_hide")]
+        public StringProperty ToolbarHide { get; set; } = "Never";
```

Comment:
```suggestion
        public StringProperty ToolbarHide { get; set; } = "When both camera and microphone are unmuted";
```

Reply:
As I understand this gives the toolbar it's the default value. Would this not make it confusing for the users if they only see the toolbar for a few seconds and then disappear until they figure out why?

Reply:
See https://github.com/microsoft/PowerToys/pull/19175#discussion_r981103389, hope it clears things up.